org.opent2t.sample.thermostat.superpopular/com.smartthings.thermostat/js/thingTranslator.js

```diff
@@ -164,7 +172,7 @@ function providerSchemaToPlatformSchema(providerSchema, expand) {
                 icv: "core.1.1.0",
                 dmv: "res.1.1.0",
                 rt: ['opent2t.d.thermostat'],
-                di: thermostatDeviceDi,
+                di: generateGUID(providerSchema['id']+'opent2t.d.thermostat'),
```

Comment:
Fixed.

---
In reply to: [108803846](https://github.com/openT2T/translators/pull/217#discussion_r108803846) [](ancestors = 108803846)